// Copyright (c) 2012 The Chromium OS Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

option optimize_for = LITE_RUNTIME;

package power_manager;

// Included in powerm's SuspendStateChanged signal.
message SuspendState {
  // Next ID to use: 3

  // Type of event.
  enum Type {
    SUSPEND_TO_MEMORY = 0;
    RESUME = 1;
  }
  optional Type type = 1;

  // Wall time at which this event occured, as given by
  // base::Time::ToInternalValue().  (base::Time is used because the monotonic
  // clock used by base::TimeTicks doesn't advance while the system is
  // suspended.)
  optional int64 wall_time = 2;
}
